sites/mundopmmi/config/site.js

```diff
@@ -43,17 +42,18 @@ module.exports = {
   ],
   menuItems: {
     resources: [
-      { href: '/magazine', label: 'Revista' },
-      { href: '/videos', label: 'Vídeos' },
-      { href: '/white-papers', label: 'Libros Blancos' },
       { href: '/events', label: 'Eventos' },
+      { href: '/', label: 'Descargas' },
+      { href: '/videos', label: 'Vídeos' },
     ],
     userTools: [
-      { href: '/subscribe', label: 'Suscribir' },
-      { href: 'https://www.pmmimediagroup.com/pw/mundo-pmmi', label: 'Anunciar', target: '_blank' },
-      { href: '/page/about-us', label: 'Sobre nosotras' },
-      { href: '/contact-us', label: 'Contáctenos' },
-      { href: '/page/privacy-policy', label: 'Política de privacidad' },
+      { href: '/suscríbase', label: 'Suscríbase' },
```

Comment:
This should likely still be `/subscribe`. Additionally for the URIs below, remove the diacritics per [RFC3986](https://tools.ietf.org/html/rfc3986#section-2) -- while browser may interpret diacritics, they ultimately strip them and make the request to the ASCII version of the URL, as URLs cannot contain diacritics.

If these are custom pages (rather than sections, for ex the privacy and conditions pages), ensure they have the `/page/` prefix.